src/model/resource_set.js

```diff
@@ -19,8 +19,130 @@
  */
 
 /**
- * @class ResourceSet is the top level interface for sets of parts of a project.
+ * @class ResourceSet is the top level interface for sets of parts of a project,
+ *     not including the project itself. Resource sets can only contain one type
+ *     of resource, and cannot contain multiple resources with the same name.
+ *
  */
  class ResourceSet {
```

Comment:
I'm guessing a ResourceSet is itself a type of Resource (it has a name, and can be loaded and saved).